### ISSUE TITLE
Typing id of a NamedArray at the prompt should display content, not pointer ref

### ISSUE THREAD

**Issue by luciansmith** [sys-bio/roadrunner#882]
In roadrunner 2.0.3, the following script:

```
r.simulate()
```
would print the results of simulating the model.  As of 2.1.1, it instead prints something like:

```
<NamedArray object at 0000021DB25F0420>
```

which is not ideal.  You can print a NamedArray:

```
print(r.simulate())
results = r.simulate()
print(results)
```
so you can still see things, but this behavior should be changed if possible.

**Comment by CiaranWelsh** [comment]
The intuition for what I did here was to take note of the differences between __repr__ and __str__, which are subtle do exist. __repr__ is supposed to be a unique representation and is used in for loops while __str__ is human readable. At any rate, I thought it was the __str__ function that would be used in the above case which is clearly wrong. We need the behaviour you described which was easily reverted back to original behaviour in 78ae2f4f2ec00ccddb2a838a44b152db35756ce2. 

Note - this was a very small change so I committed directly to develop. I've incremented to v2.1.2 and set up two pull requests one each for release and experimental release.
